From 3c55f712303350dfd696a17cf6262b2997271677 Mon Sep 17 00:00:00 2001
From: Alona Solntseva <alona.solntseva@tandemg.com>
Date: Wed, 01 Jun 2022 22:11:54 +0300
Subject: [PATCH] [SW-67516] tpm: add host-dtc to hl-smi dependency

TPM attestation functionality is added to hl-smi
It requires fit file parsing, which is supported by libfdt lib
For proper compilation host-dtc dependency is added for hl-smi

Signed-off-by: Alona Solntseva <alona.solntseva@tandemg.com>
Change-Id: I74810bcc714a39d3118085d2f275fb9c4a14f4c3
---

diff --git a/package/hl-smi/hl-smi.mk b/package/hl-smi/hl-smi.mk
index 1f23f93..0b0f9cd 100644
--- a/package/hl-smi/hl-smi.mk
+++ b/package/hl-smi/hl-smi.mk
@@ -1,7 +1,7 @@
 HL_SMI_SITE = hl-smi
 HL_SMI_LICENSE = unclear
 
-HOST_HL_SMI_DEPENDENCIES = host-eeprom-util host-pciutils host-hl-thunk host-lm-sensors
+HOST_HL_SMI_DEPENDENCIES = host-eeprom-util host-pciutils host-hl-thunk host-lm-sensors host-dtc
 
 HOST_HL_SMI_CONF_OPTS = -DTESTS_ENABLED=OFF
 HOST_HL_SMI_CONF_OPTS += -DCMAKE_EXE_LINKER_FLAGS="-static"
